docs/ref/contrib/staticfiles.txt

```diff
@@ -381,6 +381,83 @@ simply specify a custom entry in the :setting:`CACHES` setting named
     required to ensure that the file hash is correct in the case of nested file
     paths.
 
+Static Files mixins
+===================
+
+``ManifestFilesMixin``
+----------------------------
+
+.. class:: storage.ManifestFilesMixin
+
+    A mixin that can be used to provide common behavior
+    for a ``ManifestFileSystemStorage``. The ways to store and retrieve
+    the manifest file in use is configurable
+    and can be further customized by subclasses.
+
+    Here is an example using a :doc:`Custom storage system </howto/custom-file-storage>`::
+
+
+        from django.contrib.staticfiles.storage import ManifestFilesMixin
+
+        from storages.backends.s3boto import S3BotoStorage
+
+        class StaticS3Storage(ManifestFilesMixin, S3BotoStorage):
+            pass
+
+
+    **Methods and Attributes**
+
+    .. attribute:: manifest_version
+
+        The manifest format standard version.
+        By default, this is ``1.0``.
```

Comment:
I don't think this line should be broken unless you want to make it into two paragraphs, in which case you need a blank line between.

Maybe it's because I don't know that much about staticfiles, but I don't find it very easy to understand what this attribute does and how it is supposed to be used. The section below mentions that it's packed into a JSON object, but what does that do for me?